### DISCUSSION TITLE
Getting blank white screen after upgrading to Firefly Wallet 2.0.3. Unable to see my dashboard after login

### DISCUSSION THREAD

**Discussion by joreagan** [iotaledger/firefly#7754] [Help & Support]
Before upgrade i have done the backup and after migration i can login once and see the vested the IOTA in my wallet. But when i try to login again unable to see anything. Its showing blank white screen. Kindly help to resolve this issue

**Comment by begonaalvarezd** [comment]
Which exact OS and version are you on? 

In the meantime, I can recommend, If you have a backup of your seed or stronghold, to purge the firefly data, you can do so by removing the following folder and start the app again

- Windows: `%APPDATA%\Roaming\Firefly\__storage__/`
- MacOS: `$HOME/Library/Application\ Support/Firefly/__storage__/`
- Linux: `~/.config/Firefly/__storage__/`

Please let us know if this didnt resolve your issue 🤞🏼
